package.json

```diff
@@ -50,5 +50,394 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "acorn": "^7.4.1",
+    "acorn-jsx": "^5.3.1",
+    "aggregate-error": "^3.1.0",
+    "ajv": "^6.12.6",
+    "ansi-colors": "^4.1.1",
+    "ansi-escapes": "^4.3.2",
+    "ansi-regex": "^5.0.0",
+    "ansi-styles": "^4.3.0",
+    "argparse": "^1.0.10",
```

Comment:
What happened to your `code-solutions` repo's `package.json` and `package-lock.json` files? Please restore them to their original versions.